### PR TITLE
Applied edits to Kuryr content for 3.10

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up edits to https://github.com/openshift/openshift-docs/pull/10739 and https://github.com/openshift/openshift-docs/pull/5845 per [review feedback.](https://docs.google.com/document/d/1WcuZLml2tJsMKY0FGy4pzSly84Mdsztqu8omdN6O05E/edit?usp=sharing)

### CODE REVIEW

**Comment by dulek** [comment]
Actually the behavior described in https://github.com/openshift/openshift-docs/pull/8906 had changed, as we've discovered a bug with the previous approach. My bad that I haven't updated the PR. Below is the proposed update of paragraphs in line 46-55:

> {product-title} CNI will call Kuryr CNI script every time a new pod is spawned
> on or deleted from an {product-title} host. The script fetches the container ID of
> the local kuryr-cni from Docker API and executes Kuryr CNI plug-in binary through
> `docker exec` passing all the CNI call arguments. The plug-in then calls
> kuryr-daemon over local HTTP socket, again passing all the parameters.

File: architecture/topics/kuryr.adoc [hunk 1]
```diff
@@ -16,9 +16,8 @@ Kuryr SDN has some requirements regarding configuration of OpenStack it will be
 using. In particular:
 
 * Minimal service set is Keystone and Neutron.
-* A LBaaS service is required. It can be either
-  link:https://docs.openstack.org/octavia/latest/[Octavia] (with all of its own
-  dependencies) or deprecated Neutron-LBaaS v2.
+* It works with
+  link:https://docs.openstack.org/octavia/latest/[Octavia].
```
Comment (eurijon) [inline architecture/topics/kuryr.adoc]: Hey, there are two more OpenStack requirements to be added to the list:
 - Trunk Ports extension must be enabled
 - Neutron must use the Open vSwitch firewall driver
(taken from https://github.com/openshift/openshift-ansible/blob/master/playbooks/openstack/configuration.md#openstack-requirements)

**Comment by eurijon** [comment]
In line 26 of install_config/configuring_kuryrsdn.adoc:
> are installed as pods on {product-title} in the `kube-system` namespace:

it should say `openshift-infra` instead of `kube-system`.

**Comment by ahardin-rh** [comment]
@dulek @eurijon Updates complete. Thanks again!

**Review by dulek — CHANGES_REQUESTED** [review]

**Comment by ahardin-rh** [comment]
I updated the output per Jon and addressed the latest feedback from @dulek

**Comment by ahardin-rh** [comment]
@dulek @eurijon Any more updates for this, or is this okay to merge? Thanks!

cc @vikram-redhat

**Comment by eurijon** [comment]
Ok from my side

**Comment by ahardin-rh** [comment]
@eurijon I'm going to go ahead and merge this then. If further edits are needed, we can address those in a follow-up PR. Thanks!

**Pull request merged**